### PR TITLE
Amanda: Replaced info, added light/dark mode, and updated layout

### PR DESCRIPTION
**Pull request by aruan20**
<fixes L=ENG-55, ENG-57>

The placeholder information on the website was replaced with actual information for the about and project sections (more info to be added and photos will be replaced later). A button to change between light and dark mode was added to the top right of the website and the layout was updated based on Monday's UX review.

Previously:
343962393-3bb75891-eea3-47da-a639-e2debaf0a3dc

Currently:
Screenshot 2024-07-02 at 2 45 02 PM
Screenshot 2024-07-02 at 2 45 30 PM

### CODE REVIEW

File: js/src/personal/amanda/LightDarkButton.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+export function LightDarkButton() {
+  const { setColorScheme } = useMantineColorScheme()
+  const computedColorScheme = useComputedColorScheme('light')
+
+  return (
+    <Group justify="center">
+      <ActionIcon
+        onClick={() =>
+          setColorScheme(computedColorScheme === 'light' ? 'dark' : 'light')
```
Comment (HaoxinLuo) [inline js/src/personal/amanda/LightDarkButton.tsx:17]: Feel free to ignore as this is more of a fyi, but there seems to just be a `toggleColorscheme` that we can use. Not sure why the doc suggested defining our own toggle logic w/ computed color scheme.

https://mantine.dev/theming/color-schemes/#use-mantine-color-scheme-hook

File: js/src/personal/amanda/Main.page.tsx [hunk 2]
```diff
@@ -1,55 +1,133 @@
...
+          <Contact />
+          <Space h="xl" />
+          <Navbar />
+        </Container>
+        <ScrollArea
+          className={styles.content}
+          type="scroll"
+          offsetScrollbars
+        >
+          {info.map((section) => (
```
Comment (HaoxinLuo) [inline js/src/personal/amanda/Main.page.tsx:57]: nit: if we are using .map with .map with .map, it seems like a good candidate to refactor into smaller components

File: js/src/personal/amanda/Landing.page.tsx [hunk 3]
```diff
@@ -1,10 +1,14 @@
...
 import { MouseEventHandler } from 'react'
 import styles from './Amanda.module.css'
+import { LightDarkButton } from '@/personal/amanda/LightDarkButton.tsx'
 
 export function LandingPage({ onClick }: { onClick: MouseEventHandler }) {
   return (
     <>
+      <Container className={styles.lightDark}>
+        <LightDarkButton />
+      </Container>
```
Comment (HaoxinLuo) [inline js/src/personal/amanda/Landing.page.tsx:11]: nit: make a top "nav bar" that is shared across landing and Main page so the lightdarkbutton is shared and used once.

File: js/src/personal/amanda/Types.tsx [hunk 4]
```diff
@@ -1,6 +1,8 @@
-interface Project {
+export interface Project {
```
Comment (arklian) [inline js/src/personal/amanda/Types.tsx:1]: nit: After reading about it a bit, I think `type` is likely better than `interface` for us, but both work.

**Review by arklian — APPROVED** [review]
Nice!

**Pull request merged**